src/Product.php

```diff
@@ -421,20 +421,20 @@ public function processVariation(array $variation)
             return false;
         }
 
-        if ($variation['isMain'] || $this->getField('sort') === '') {
-            $this->setField('sort', $this->getFromArray($variation, 'position'));
+        if ($this->getField('sort') === '') {
+            $this->setField('sort', $this->getFromArray($variation['base'], 'position'));
```

Comment:
Is `base` now the same as `isMain`?

Reply:
No base contains all data of the base product. But it is also merged with some information about the current variation. `$variation['base']['isMain']` is the same as before. This is also how it was implemented in 263.